tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-ci-pipeline.yml

```diff
@@ -16,7 +16,6 @@ pr:
   branches:
     include:
     - main
-    - rel-*
```

Comment:
The pipeline is a required pipeline. You still need to run the pipeline so that your PR can be merged. However,you may empty out the most content of this yaml file.